### ISSUE TITLE
Killmoves don't work properly in third person.

### ISSUE THREAD

**Issue by Hashigo** [ShikyoKira/Project-New-Reign---Nemesis-Main#140]
Oftentimes, killmoves play only for the user in third person, and NPC's don't respond nor do they die.

**Comment by ShikyoKira** [comment]
are you using ultimate combat? If so you have to turn of npc ai

**Comment by ShikyoKira** [comment]
are you sure you are using the latest version?

**Comment by Hashigo** [comment]
Yes, I've been using the latest version. Though I should have mentioned that this effect was happening while playing through the Skigerfall mod (The author of this mod refereed me to you) with character animations enhanced. No custom body mods are being used. The issue happens during every killmove.

**Comment by ShikyoKira** [comment]
Do you have link to the mod? I can't find it. Maybe my google skill is just bad

Have you tried this with a clean minimal load order to recreate this issue? Just wanna rule out the possibility of mod conflict with other mods as well not just nemesis

**Comment by Hashigo** [comment]
Here's a Nexus link to the mod: https://www.nexusmods.com/skyrim/mods/99469

I'll try it with a smaller load order when I'm able, however, every mod on the load order are, for the most part, animation mods and Character Behaviors enhanced

**Comment by ShikyoKira** [comment]
remove CBE and test it again
